app/controllers/map_controller.rb

```diff
@@ -7,15 +7,15 @@ def index
   def favorite
     @favorite = Favorite.new
     @favorite.user_id = set_current_user.id
```

Comment:
This is causing an error if there is not user signed in. Maybe only show the star if they are signed in?